packages/fxa-settings/src/models/hooks.ts

```diff
@@ -82,3 +82,25 @@ export function useFtlMsgResolver() {
   const { l10n } = useLocalization();
   return new FtlMsgResolver(l10n, config.l10n.strict);
 }
+
+/**
+ * Hook to run a function on an interval.
+ * @param callback - function to call
+ * @param delay - interval in Ms to run, null to stop poll
+ */
+export function useInterval(callback: () => void, delay: number | null) {
+  const savedCallback = useRef(callback);
```

Comment:
Does this mean the savedCallback is destroyed when the compoenent is destroyed / unmounted?

Reply:
Yea that was my understanding.